cmd/ocm-load-test.go

```diff
@@ -86,28 +85,37 @@ func initConfig() {
 }
 
 func run(cmd *cobra.Command, args []string) error {
-	if viper.GetString("ocm-token") == "" {
-		return fmt.Errorf("ocm-token is a necessary configuration")
+	logger, err := logging.NewGoLoggerBuilder().
+		Debug(viper.GetBool("verbose")).
+		Build()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't build logger: %v\n", err)
+		os.Exit(1)
 	}
 
-	err := helpers.CreateFolder(viper.GetString("output-path"))
+	if viper.GetString("ocm-token") == "" {
+		logger.Fatal(cmd.Context(), "ocm-token is a necessary configuration")
+	}
+	err = helpers.CreateFolder(viper.GetString("output-path"), logger)
 	if err != nil {
-		return fmt.Errorf("creating output directory: %s", err)
+		logger.Fatal(cmd.Context(), "creating api connection: %v", err)
 	}
-	log.Printf("Using output directory: %s", viper.GetString("output-path"))
+	logger.Info(cmd.Context(), "Using output directory: %s", viper.GetString("output-path"))
 
 	connection, err := helpers.BuildConnection(viper.GetString("gateway-url"),
 		viper.GetString("client.id"),
 		viper.GetString("client.secret"),
-		viper.GetString("ocm-token"))
+		viper.GetString("ocm-token"),
+		logger,
+		cmd.Context())
 	if err != nil {
-		return fmt.Errorf("creating api connection: %v", err)
+		logger.Fatal(cmd.Context(), "creating api connection: %v", err)
```

Comment:
Same as the other question -- should this be Fatal?

Reply:
This one should, we cannot do anything if creating the connection fails.

Reply:
That makes sense!